apps/client/src/features/viewers/minimal-timer/MinimalTimer.tsx

```diff
@@ -135,8 +135,8 @@ export default function MinimalTimer(props: MinimalTimerProps) {
   const showFinished = finished && !userOptions?.hideOvertime && (time.timerType !== TimerType.Clock || showEndMessage);
 
   const showProgress = time.playback !== Playback.Stop;
-  const showWarning = (time.current ?? 1) < viewSettings.warningThreshold;
-  const showDanger = (time.current ?? 1) < viewSettings.dangerThreshold;
+  const showWarning = (time.current ?? 1) < (time.timeWarning ?? 0);
+  const showDanger = (time.current ?? 1) < (time.timeDanger ?? 0);
```

Comment:
would this not meant that 
if the timer is not running and the user hasnt set a warning timer, we are in `showWarning`?

I wonder if the logic here can be simplified

Reply:
Shouldn't this come out as false if there is no timeWarning? Since time.current isn't smaller that timeWarning then. Or in other words, we compare if time.current is smaller than 0 so we won't ever go into showWarning. The only time when this would be true is if the timer goes negative, which is when we are in the red overtime anyway.

Reply:
timer is loaded (not playing) and there is no `timeWarning`

```ts
const showWarning = (time.current ?? 1) < (time.timeWarning ?? 0);
const showWarning = (null ?? 1) < (null ?? 0);
const showWarning = (1) < (0);
const showWarning = false;
```

timer is loaded (not playing) and there is `timeWarning`

```ts
const showWarning = (time.current ?? 1) < (time.timeWarning ?? 0);
const showWarning = (null ?? 1) < (6000 ?? 0);
const showWarning = (1) < (6000);
const showWarning = true;
```

Reply:
This logic checks out. But if a timer is loaded we already have the time.current information, which is at that point the starting time of the timer

Reply:
Sorry, you are correct. I was under the impression that loaded events had null current time payload, but that is not the case